### ISSUE TITLE
import name 'SyncBN' from 'utils.syncbn' (unknown location)

### ISSUE THREAD

**Issue by ks322181** [Zhongdao/Towards-Realtime-MOT#76]
cannot import name 'SyncBN' from 'utils.syncbn' (unknown location)

I compiled and placed it under utils/syncbn. But still facing the same issue

Need help..

Thanks in Advance

**Comment by HuangXuan1996** [comment]
I meet the same question,do you solve it?

**Comment by HuangXuan1996** [comment]
(Optional) syncbn (compile and place it under utils/syncbn, or simply replace with nn.BatchNorm here)
download

**Comment by ks322181** [comment]
yaa i have done. But still facing the same issue

**Comment by Zhongdao** [comment]
In the current version this issue is solved by  try/except, see [here](https://github.com/Zhongdao/Towards-Realtime-MOT/blob/a340fdeccdf7fe9dc6381d0efe7091631a439ac6/models.py#L11).
I think this issue could be closed, pls re-open it if you still have problems.

**Comment by ks322181** [comment]
It resolved my issue, but now i want to train my own model so is it fine to use nn.batchnorm instead of syncbn. Because here you mentioned to train with syncbn results will be https://github.com/Zhongdao/Towards-Realtime-MOT/issues/23#issuecomment-542024827
